web_poet/page_inputs/browser.py

```diff
@@ -7,7 +7,7 @@
 from .url import ResponseUrl
 
 
-class BrowserHtml(SelectableMixin, str):
+class BrowserHtml(SelectableMixin, str):  # noqa: SLOT000
```

Comment:
Not 100% sure how it works, but it looks like even if `__slots__` is defined here, it has no effect (I assume because the other base class doesn't have it).